scripts/ci_setup.py

```diff
@@ -233,10 +233,12 @@ def __main(args: list) -> int:
                 out_file.write(variable_format % ('PERFLAB_BUILDTIMESTAMP', source_timestamp))
                 out_file.write(variable_format % ('PERFLAB_CONFIGS', config_string))
                 out_file.write(variable_format % ('DOTNET_VERSION', dotnet_version))
+                out_file.write(variable_format % ('PYTHON_SCRIPT_TARGET_FRAMEWORKS'), framework)
 
         else:
             with open(args.output_file, 'w') as out_file:
                 out_file.write(variable_format % ('PERFLAB_INLAB', '0'))
+                out_file.write(variable_format % ('PYTHON_SCRIPT_TARGET_FRAMEWORKS'), framework)
```

Comment:
I like this - we have other tools that will fall into this problem. If we're just going to set this globally, can we do a slightly larger change and just rename the variable to `PERFLAB_TARGET_FRAMEWORKS`? The `benchmarks_ci.py` script can still set/use it.

Reply:
Would you want it changed to PERFLAB_TARGET_FRAMEWORKS everywhere, or just in this case?

Reply:
Everywhere. (Which is really just a few more `csproj` files and the python script that sets it, I think.)

Reply:
Cool.

Reply:
I updated it in MicroBenchmarks.csproj, and added it to Roslyn and ML. If Roslyn/ML fail, I'll probably back out that change.

Reply:
Fair!